### ISSUE TITLE
[add dataset] Kissling_2019: Processing submission

### ISSUE THREAD

**Issue by ehwenk** [traitecoevo/austraits.build#420]
**Source**: Kissling et al (2019) PalmTraits 1.0, a species-level functional trait database of palms
worldwide *Scientific Data* http://doi.org/10.1038/s41597-019-0189-0
**Custodian**: Daniel Kissling <wdkissling@gmail.com>
**Handled by**: Lizzy Wenk

- [x] Contact made with contributor (label as `waiting on response` if waiting for files)
- [x] Files received from contributor saved in google drive folder `austraits/data submissions/new`  ( label as `ready` )
- [x] Started importing files into own Austraits branch  (mark label as `In progress` and assign issue to relevant person )
- [x] Metadata complete, builds into Austraits
- [x] Report reviewed by AusTraits team
- [ ] Report sent to contributor ( label as `waiting on response` )
- [ ] Response received from contributor
- [ ] Files updated & report finalised
- [ ] Final checks ( all tests pass )
- [ ] Branch merged into master branch (issue `closed` )

NB: 

- if you need to change the config files, e.g. to add a unit conversion or define a new trait. This should be done on the master branch (via pull or rebase)
